src/components/Router/StepComponentMap.js

```diff
@@ -39,14 +40,29 @@ export const enabledDocuments = (steps) => {
   return docTypes ? Object.keys(docTypes).filter((type) => docTypes[type]) : []
 }
 
+const getFaceStep = (steps) => {
+  if (shouldUseVideo(steps)) {
+    return [VideoCapture, VideoConfirm]
+  } else {
+    return getSelfieStep(steps)
+  }
```

Comment:
This is breaking the build because of an eslint rule where the `else` statement is redundant if `return` is used inside the `if` statement. To fix it, remove the `else` statement
```
if (shouldUseVideo(steps)) {
    return [VideoCapture, VideoConfirm]
}
return getSelfieStep(steps)
```